client/connect-account-page/utils.ts

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
Yes, I put it there because it's only used by connect account page. But we could definitely use it in other places too. So I'll move it 🙂 Addressed in f8ae56a.